model/components/jaeger.yml

```diff
@@ -0,0 +1,52 @@
+---
+components:
+
+  # jaeger agent
+  - id: jaeger-agent
+    name: Jaeger Agent
+    description: The Jaeger agent is a network daemon that listens for spans sent over UDP, which it batches and sends to the collector. It is designed to be deployed to all hosts as an infrastructure component. The agent abstracts the routing and discovery of the collectors away from the client.
+    link: http://www.jaegertracing.io/
+    logo: jaeger
+    categories:
+      - agent
+    capabilities:
+      - tech:
+        - java
+        - go
+        - nodejs
+        - python
+        - c++
+      - aspects:
+        - tracing
+      - frameworks:
+        - spring
+        - jdbc
+        - ldap
+
+  # jaeger Collector
+  - id: jaeger-collector
+    name: Jaeger Collector
+    description: The Jaeger collector receives traces from Jaeger agents and runs them through a processing pipeline. Currently our pipeline validates traces, indexes them, performs any transformations, and finally stores them.
+    link: http://www.jaegertracing.io/
+    logo: jaeger
+    categories:
+      - collector
+    connections:
+      - cassandra
+      - elasticsearch
+      - memory
```

Comment:
I think we can remove memory from here, since otherwise we would need to provide a corresponding "memory" component in the model.